src/artm/score/perplexity.cc

```diff
@@ -66,7 +66,7 @@ void Perplexity::AppendScore(
       }
     }
     if (class_weight_map.empty()) {
-      LOG(ERROR) << "None of requested classes were presented in model. Score calculation will be skipped";
+      LOG_FIRST_N(ERROR, 100) << "None of requested classes were presented in model. Score calculation will be skipped";
```

Comment:
Very optional, but maybe replace 100 with some parameter?

Reply:
I'd keep explicit 100 here - there is no need to have consistent number LGO_FIRST_N threshold across library.